CenterofExcellenceAuditLogs/SolutionPackage/environmentvariabledefinitions/admin_TenantID/environmentvariabledefinition.xml

```diff
@@ -1,12 +0,0 @@
-<environmentvariabledefinition schemaname="admin_TenantID">
```

Comment:
Just checking this was intentionally deleted. Don't see it referenced anywhere else in the PR. Was it just an unused variable that needed cleaned up?

Reply:
This has also moved to Core now!